projects/core/src/occ/occ-models/occ.models.ts

```diff
@@ -4288,7 +4280,7 @@ export namespace Occ {
     code?: string;
     customerOrderApprovalRecords?: OrderApprovalRecord[];
     merchantOrderApprovalRecords?: OrderApprovalRecord[];
-    order?: B2BOrder;
```

Comment:
Don't know if it's needed in the my-company counterpart, but we lost chinesePaymentInfo, paymentStatus, totalUnitCount property, so maybe we need to add it to the order model